doozerlib/distgit.py

```diff
@@ -906,8 +907,37 @@ def _build_container(
         self.logger.info("Successfully built image: {} ; {}".format(target_image, task_url))
         return True
 
-    def _logs_dir(self):
-        return "%s/%s" % (self.runtime.brew_logs_dir, self.metadata.name)
+    def _logs_dir(self, task_id=None):
+        segments = [self.runtime.brew_logs_dir, self.metadata.name]
+        if task_id:
```

Comment:
I think `pep` would tell you `if task_is is not None`, but whatever.

Reply:
@tbielawa no, this is valid as long as you really are just looking at the "truthiness" of the variable. 
I agree that, for consistency if should be `if task_id is not None` because that's the default param value given. If you are OK with any value where `bool(task_id) is False` then this is more concise and I'm OK with it.

Reply:
I went ahead and put `is not None` in there, as the intent is arguably clearer that way.